eta/core/video.py

```diff
@@ -1639,6 +1669,9 @@ def __init__(
             inpath: path to the input video. Passed directly to a VideoReader
             frames: an optional range(s) of frames to process. This argument
                 is passed directly to VideoReader
+            keyframes_only: whether to only extract keyframes when reading the
```

Comment:
I clarified that you can use `frames` and `keyframes_only` together, which is cool!